certification/engine/engine.go

```diff
@@ -158,3 +170,7 @@ func ContainerPolicy() []string {
 func ScratchContainerPolicy() []string {
 	return makeCheckList(scratchContainerPolicy)
 }
+
+func PrivilegedContainerPolicy() []string {
```

Comment:
Also, let's not name this "Privileged". They are overloading it in Pyxis, but this has a completely different connotation. We are actually checking running as root, not for --privileged. They have different meanings. I think just RootExceptionContainerPolicy should do the trick.

Reply:
Change the name to the one suggested above.